pyk4a/playback.py

```diff
@@ -168,7 +168,7 @@ def get_next_capture(self):
             thread_safe=self.thread_safe,
         )
 
-    def get_previouse_capture(self):
+    def get_previous_capture(self):
```

Comment:
Maybe something like this? 

```suggestion
   get_previouse_capture(self):
       warnings.warn("get_previouse_capture() deprecated, please use get_previous_capture() instead", DeprecationWarning)
       return self.get_previous_capture() 

    def get_previous_capture(self):
```

Reply:
Sure! That's much cleaner, sorry I'm not used to this process.